lib/OpenQA/WebAPI/Controller/API/V1/Job.pm

```diff
@@ -102,9 +102,9 @@ sub list {
 
     my %args;
     $args{limit} = $limit;
-    my @args = qw(build iso distri version flavor maxage scope group
-      groupid page before after arch hdd_1 test machine worker_class
-      failed_modules modules modules_result);
+    my @args = qw(build iso distri version flavor scope group groupid page
```

Comment:
Looks like you now also got rid of `maxage`. Is it really useless to be able to filter out old jobs?

Reply:
Yes. Is it used somewhere?

Reply:
You'll have to ask our users. (This is not an internal API.)

Reply:
I already asked and nobody responded. Can we please expect the missing removal to be an oversight of https://github.com/os-autoinst/openQA/commit/4865792863e7f298a23b3640052c748b05d265e0 and if someone complains we will add it back?

Reply:
Same reasoning as for `failed_modules` applies for `maxage`, see https://github.com/os-autoinst/openQA/pull/3918/commits/f055c31b6cc8744bfe066474a70d3877dc08bfb1 . It's undocumented, untested, not used
anywhere within openQA and very likely not used anywhere outside the
project scope as well as no one ever again reported an issue about that, asked a question or brought it up in any context.

Reply:
Should be groupids (plural) in this list too?